res/css/views/rooms/_IRCLayout.scss

```diff
@@ -198,6 +198,10 @@ $irc-line-height: $font-18px;
 
         > .mx_SenderProfile_displayName {
```

Comment:
How would this work for disambiguated users (users that share the same display name, so we also show their `mxid`)?